src/js/components/AppPageComponent.jsx

```diff
@@ -439,12 +440,14 @@ var AppPageComponent = React.createClass({
       content = this.getTaskDetailComponent();
     }
 
+    var group = ViewHelper.getGroupFromAppId(state.appId);
+
     return (
       <div>
-        <AppBreadcrumbsComponent
-          activeTaskId={state.activeTaskId}
-          activeViewIndex={state.activeViewIndex}
-          appId={state.appId} />
+        <BreadcrumbComponent
```

Comment:
tiny little nitpick – didn't we say first JSX argument should be inline with the component?

```
<BreadcrumbComponent app={state.appId}
  group={group}
  task={state.activeTaskId} />
```